### PR TITLE
Fix formatting issue. Add Raspberry PI jni lib.

### PR DESCRIPTION
**Pull request by agasser**
Fix improperly formatted text in example.
Add Raspberry PI JNI library. The comment says it is adding all supported platforms. Additionally Raspberry Pi is a good use case for a standalone NetworkTables client.

### CODE REVIEW

File: source/docs/software/networktables/client-side-program.rst [hunk 1]
```diff
@@ -10,7 +10,9 @@ A basic client program looks like the following example.
 .. tabs::
 
    .. code-tab:: java
+
       package networktablesdesktopclient;
+
```
Comment (sciencewhiz) [inline source/docs/software/networktables/client-side-program.rst:15]: Fix trailing white space
```suggestion

```

**Review by sciencewhiz — CHANGES_REQUESTED** [review]

File: source/docs/software/networktables/client-side-program.rst (hunk 1 above)
Comment (agasser) [reply source/docs/software/networktables/client-side-program.rst]: Fixed

**Comment by Daltz333** [comment]
Should the comment get removed?

**Comment by agasser** [comment]
> Should the comment get removed?

That's a good question. It is helpful to have all of the environments in the example, otherwise another section should be added to explain. In my use case I was using a Raspberry Pi and I had to solve it myself when I got an error attempting to run it.

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**